plugins/avx512/int/UMESimdVecInt64_16.h

```diff
@@ -2371,8 +2337,8 @@ namespace SIMD {
 #else
             __m512i t3 = _mm512_setr_epi64(0, stride, 2*stride, 3*stride, 4*stride, 5*stride, 6*stride, 7*stride);
             __m512i t4 = _mm512_setr_epi64(8*stride, 9*stride, 10*stride, 11*stride, 12*stride, 13*stride, 14*stride, 15*stride);
-#endif
-#if defined(WA_GCC_INTR_SUPPORT_6_4)
+#endif
+#if defined(WA_GCC_INTR_SUPPORT_6_4) || defined(WA_GCC_INTR_SUPPORT_7_1)
```

Comment:
Whatever applies to GCC with higher major version, also applies to GCC with lower major version number. This line can be safely replaced with:

#if defined(WA_GCC_INTR_SUPPORT_7_1)

Reply:
i relooked how you define the macros.. now i understand. but then the order is important. so you always define first the code for ..SUPPORT_6_4 and after this ..SUPPORT_7_1 if there are different versions of code possible.